src/harness/BenchmarkDotNet.Extensions/Extensions.cs

```diff
@@ -17,10 +17,20 @@ public static int ToExitCode(this IEnumerable<Summary> summaries)
                 return 1;
 
             // if anything has failed, it's an error
-            if (summaries.Any(summary => summary.HasCriticalValidationErrors || summary.Reports.Any(report => !report.BuildResult.IsBuildSuccess || !report.AllMeasurements.Any())))
+            if (summaries.Any(summary => summary.HasAnyErrors()))
```

Comment:
Does this widen the meaning of what we consider a failure from before? I am just not familiar with this method.

Reply:
no, the method is implemented just below it, I just refactored the code a bit so that I could use it from the reporter code